src/engine/engine.cpp

```diff
@@ -662,6 +663,21 @@ namespace CityFlow {
         return step * interval;
     }
 
+    double Engine::getAverageTravelTime() const {
+        double tt = 0;
+        int n = 0;
+        for (auto &vehicle_pair : vehiclePool) {
+            auto &vehicle = vehicle_pair.second.first;
+            tt += getCurrentTime() - vehicle->getEnterTime();
+            n++;
+        }
+        for (double tt_ : finishedVehicleTravelTime) {
+            tt += tt_;
+            n++;
+        }
+        return n == 0 ? 0 : tt / n;
+    }
+
```

Comment:
I used a brute force method to compute average tt, which might have high cost when there are lots of finished vehicles. Could you please optimize it?